ODIN_II/SRC/verilog_bison.y

```diff
@@ -393,13 +396,22 @@ statement:
 	| ';'																						{$$ = NULL;}
 	;
 
-list_of_specify_statement:
-	list_of_specify_statement specify_statement 						{$$ = newList_entry($1, $2);}
-	| specify_statement							                        {$$ = newList(SPECIFY_PAL_CONNECT_LIST, $1);}
-    ;
+list_of_specify_items:
+	list_of_specify_items specify_item 							{$$ = newList_entry($1, $2);}
+	| specify_item										{$$ = newList(SPECIFY_ITEMS, $1);}
+	;
+
+specify_item:
+	specify_pal_connect_declaration            						{$$ = newList(SPECIFY_PAL_CONNECT_LIST, $1);}
+	| specparam_declaration            						        {$$ = cleanup_specify_items($1);}
+	;
+
+specify_pal_connect_declaration:
+	'(' parallel_connection ')' '=' primary ';'						{$$ = newParallelConnection($2, $5, yylineno);}
+	;
 
-specify_statement:
-    '(' parallel_connection ')' '=' primary ';'							{$$ = newParallelConnection($2, $5, yylineno);}
```

Comment:
is the newParallelConnection function used anywhere else? if not, please remove it

Reply:
It seems that the `newParallelConnection` function is being used by `specify_pal_connect_declaration`. It was not added in this PR btw.

Reply:
my bad! just noticed, thanks